docs-src/docs/dataset-exploration/uploading-predictions.md

```diff
@@ -56,6 +58,23 @@ except requests.exceptions.RequestException as e:
     print(f"Request error: {e}. {msg}")
 ```
 
+Predictions groups connected to segmentation datasets require one extra parameter called `classMapping`. This is used to map the class names 
+in the predictions to the class names in the annotations. The `classMapping` parameter is a list of dictionaries, where each dictionary
```

Comment:
I think we could be even more clear that this affects the matching and what will be marked as disagreement in the gallery.

Reply:
Good point, will fix